src/adb/forward.js

```diff
@@ -0,0 +1,19 @@
+let debug = console.log.bind(console, '[adb/forward]');
+let exec = require('mz/child_process').exec;
```

Comment:
Yeah I totally agree, I'm just thinking about when I'll add `-H`, `-P`, and `--serial`, these will need to be passed into every adb `exec` call, so it would be nice if we didn't duplicate that logic everywhere. But for now, we don't need to worry.